utils/common.go

```diff
@@ -21,3 +21,15 @@ func GetTestLogger(t *testing.T) zerolog.Logger {
 	l := zerolog.New(zerolog.NewTestWriter(t)).Output(zerolog.ConsoleWriter{Out: os.Stderr}).Level(lvl).With().Timestamp().Logger()
 	return l
 }
+
+// GetTestComponentLogger instantiates a logger that takes into account the test context, log level and a component name
+func GetTestComponentLogger(t *testing.T, name string) zerolog.Logger {
```

Comment:
Just wondering how this differs from our current `GetLogger`?

Reply:
I can do it with `functional options` and have one function. Basically, it's adding a `component` field. But I'm not sure you need it.

Reply:
Okay, can skip this function and just do
```
	l := utils.GetTestLogger(t).With().Str("Component", "LogWatch").Logger()
```